engine/src/main/java/org/terasology/rendering/dag/nodes/DirectionalLightsNode.java

```diff
@@ -152,7 +153,7 @@ private void applyLightBufferPass() {
         bindDisplay();     // TODO: verify this is necessary
         setViewportToSizeOf(sceneOpaque);    // TODO: verify this is necessary
 
-        frameBuffersManager.swapSceneOpaqueFBOs();
+        dynamicFBM.swap(DefaultDynamicFBOs.WriteOnlyGBuffer.getResourceUrn(), DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
So how should we move on? By just adding notification if there is a swap or limiting swapping behaviour only for those two?